README.md

```diff
@@ -1,10 +1,27 @@
-# Markdown Editor
+<h1 align="center">
+  Markdown Editor
+</h1>
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/952fdc5d-a2bb-4895-a375-25ea1c6f30d8/deploy-status)](https://app.netlify.com/sites/accordproject-markdown-editor/deploys) 
-[![downloads](https://img.shields.io/npm/dm/@accordproject/markdown-editor)](https://www.npmjs.com/package/@accordproject/markdown-editor)
-[![GitHub license](https://img.shields.io/github/license/accordproject/markdown-editor?color=bright-green)](https://github.com/accordproject/markdown-editor/blob/master/LICENSE)
-[![npm version](https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor.svg)](https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor)
-[![join slack](https://img.shields.io/badge/Slack-Join%20Slack-blue)](https://accord-project-slack-signup.herokuapp.com/)
+
+
+<p align="center">
+
+
+   <a href="https://app.netlify.com/sites/accordproject-markdown-editor/deploys">
```

Comment:
I believe this could be changed to `https://accordproject-markdown-editor.netlify.com/examples/`

Reply:
Yes, ma'am.

> One change on a hyperlink.

Yes, ma'am

Reply:
Yes, ma'am.